worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1732,6 +1735,85 @@ public int makeCylinder(BlockVector3 pos, Pattern block, double radiusX, double
         return affected;
     }
 
+    /**
+     * Makes a splatter.
+     *
+     * @param pos Center of the splatter
+     * @param block The block pattern to use
+     * @param radius The cylinder's radius
+     * @param decay The higher the number the more decay, between 0 and 10
+     * @return number of blocks changed
+     * @throws MaxChangedBlocksException thrown if too many blocks are changed
+     */
+    public int makeSplatter(BlockVector3 pos, Pattern block, double radius, int decay) throws MaxChangedBlocksException {
+        //  decay is 0 - 10, 0 should be a circle.
+        checkArgument(decay >= 0, "decay >= 0");
```

Comment:
This should probably be something like `"decay must be greater than or equal to 0"`, or just `"decay must be >= 0"`.

Similar for the line below.

Reply:
Sure thing - I just copied the pattern from other functions like `fillXZ` and `removeAbove`